stempy/image.cpp

```diff
@@ -578,6 +592,81 @@ RadialSum<uint64_t> radialSum(InputIt first, InputIt last, int scanWidth, int sc
   return radialSum;
 }
 
+namespace {
+// helper function that returns the binding result
+std::vector<float> getContainer(const STEMImage& inImage, const int numBins)
+{
+  // find the min and max value across all the input STEM images
+  auto min = std::numeric_limits<float>::max();
+  auto max = std::numeric_limits<float>::min();
+
+  // information about input STEMImage
+  int width = inImage.width;
+  int height = inImage.height;
+  auto curData = inImage.data;
+  std::cout << "Input STEM Image has width = " << width
+            << ", height = " << height << std::endl;
+
+  // curImage.data is shared_ptr<uint64_t []>
+  for (int i = 0; i < width * height; i++) {
+    // std::cout << "curData[" << i << "] = " << curData[i] << std::endl;
+    if (curData[i] < min)
+      min = curData[i];
+    if (curData[i] > max)
+      max = curData[i];
+  }
```

Comment:
By the way, I don't think this needs to be changed for this commit, but for future reference, it's good to use STL algorithms where possible, such as `std::minmax_element`. I believe that could be used here like so:
```
auto result = std::minmax_element(curData.get(), curData.get() + width * height);
uint64_t min = *result.first;
uint64_t min = *result.second;
```

Reply:
Thank you very much for including the actual code, I've changed it to STL algorithm.